streams/lms_metadata.py

```diff
@@ -48,21 +53,22 @@ def connect(self):
             self.connected = True
           else:
             time.sleep(0.1)
-      except:
+      except Exception as e:
         # When first creating an LMS stream, there can be random errors that will close the while loop
         # typically when asking the player for info when there isn't a player linked to the stream yet
-        print("FAIL")
+        print(f"FAIL: {e}")
```

Comment:
To me FAIL would indicate some fatal error that would make this script crash. Is there a more useful message? Should this only print when the debug flag is passed?